setup.cfg

```diff
@@ -1,7 +1,6 @@
 [metadata]
 name = mitba
 classifiers =
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3.4
     Programming Language :: Python :: 3.5
     Programming Language :: Python :: 3.6
```

Comment:
@maormarcus In the "extras" section there are package installations which depend on python 3 or not. Please fix it as well